docs/root/intro/deprecated.rst

```diff
@@ -39,6 +39,8 @@ Version 1.12.0 (pending)
   has been deprecated in favor of the `traffic_direction` field in
   :ref:`Listener <envoy_api_msg_Listener>`. The latter takes priority if
   specified.
+* Use of google.protobuf.Struct for extension opaque configs is deprecated. Use google.protobuf.Any instead or pack
+  udpa.type.v1.TypedStruct in google.protobuf.Any.
```

Comment:
Are you planning on implementing this in this PR? We should probably have the new solution implemented before deprecating :) Otherwise, yeah, we're heading in the right direction here.

Reply:
Let's do that in a separate PR and merge that one before this.

Reply:
PR #8515